packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/getSourceCodeFromMessage..st

```diff
@@ -0,0 +1,16 @@
+code browser
+getSourceCodeFromMessage: aMessageTally
+
+	| source class selector|
+	
+	(self checkIfClassExists: aMessageTally)
+	ifFalse: [^ 'Missing'].
+	
+	class := self getClass: aMessageTally.
+	selector := aMessageTally method selector.
+	source := (class == UndefinedObject and: [selector == #DoIt]) 
+		ifTrue: ['DoIt', String cr, String cr, 'Missing']
+		ifFalse: [class sourceMethodAt: selector ifAbsent:['Missing']].
```

Comment:
Please consider moving `class sourceMethodAt: selector ifAbsent:['Missing']` to a well-named function.

Reply:
Meiner Meinung ist auch hier ersichtlich was im Code passiert. Durch `sourceMethodAt:` ist hier gut zu erkennen, dass der SourceCode gesucht ist.

Reply:
Natürlich kann mensch aus dem Code erkennen, was passiert. Es ist aber deutlich einfacher, einen gut benannten Methodennamen zu lesen, als einen Ausdruck mit mehreren Methodenaufrufen und einem Magic String.